### DISCUSSION TITLE
Is it possible to modify toString() method in ParametrizedInput model?

### DISCUSSION THREAD

**Discussion by skataryna** [kobylynskyi/graphql-java-codegen#881] [Q&A] [answered]
Currently, `toString()` method looks like this:
```
public String toString() {
        StringJoiner joiner = new StringJoiner(", ", "(", ")");
        if (id != null) {
            joiner.add("id: " + GraphQLRequestSerializer.getEntry(id));
        }
        return joiner.toString();
    }
```

In our project it's possible to send `null` as an `id` parameter.
So, do your framework allow to ~~do magic~~ use some parameter to generate `toString()` which will looks like this:
```
public String toString() {
        StringJoiner joiner = new StringJoiner(", ", "(", ")");
        joiner.add("id: " + GraphQLRequestSerializer.getEntry(id));
        return joiner.toString();
    }
```

Thanks

**Comment by kobylynskyi** [comment, marked as answer]
This is not possible with the current implementation.

In the future releases we could consider adding some additional configuration parameter (we could name it `includeNullValuesInRequest` or `includeNullValuesInToString` with default value `false`).
Please create a feature request for this and hopefully someone might pick it up!
